webpack-demo/src/index.js

```diff
@@ -1,23 +1,58 @@
 /*eslint-disable*/
 
-import _ from 'lodash';
-import printMe from './print.js';
-//import './style.css';
+//import _ from 'lodash';
+import './style.css';
 
-function component() {
-  const element = document.createElement('div');
-  const btn = document.createElement('button');
+class Activity {
+  constructor(description) {
+    this.index = 0;
+    this.completed = false;
+    this.description = description;
+  }
```

Comment:
- [ ] Good job creating the Activity class for creating tasks 👏 , but as required you should create a array that contains all the tasks to be rendered, note that the method you use is another way to do that, but to use a array will give more dynamism to your code 😄 , what do you think of the code below 👍 
```bash
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
```

Reply:
Thanks @iambenkis, I appreciate your kind suggestion!.